fast-vm

```diff
@@ -907,10 +925,11 @@ case "$1" in
 						pmsg $P_ERROR "failed to start VM $vm_name, check syslog for more information\n" $vm_number
 					else
 						wait_for_ssh "$vm_number"
+						create_sshkey_if_needed
```

Comment:
I don't like that SSH key will get generated just because we have done SSH to machine. Description of commit says `fast-vm ssh changes to utilize the fast-vm ssh key if it exists` which to me sounds like the SSH key is used only when found, not that it is automatically created. I would prefer the behaviour described in the commit message - in short it means that SSH key is generated only when needed - here it will be only in the `keydist` action.

Reply:
Ah, I misinterpreted your earlier response, sorry.  In this version it is only created when when you call keydist and it doesn't already exist.